etc/Cyberfox.profile

```diff
@@ -2,5 +2,4 @@
 # This file is overwritten after every install/update
 
 
-# Redirect
```

Comment:
Add it back.  I need "redirect" as a marker to add new commands globally, and some redirected profiles are not aliases. For example thunderbird redirects to firefox. I'll be bringing in some more, thanks!